package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "server": "nodemon index.js",
-    "start": "node index.js",
+    "migration": "npx knex migrate:down --env=production && npx knex migrate:latest --env=production",
+    "start": "npm run migration node index.js",
```

Comment:
Let's keep `npm start` for development machine. so, we can run different migrations for different environment. Let's create another script tag for production. 'npm build`